src/Events/Admin/Onboarding/Controller.php

```diff
@@ -76,7 +78,37 @@ public function add_actions(): void {
 		add_action( 'rest_api_init', [ $this, 'register_rest_endpoints' ] );
 		add_action( 'admin_post_' . Landing_Page::DISMISS_ONBOARDING_PAGE_ACTION, [ $this, 'handle_onboarding_page_dismiss' ] );
 		add_action( 'admin_notices', [ $this, 'remove_all_admin_notices_in_onboarding_page' ], -1 * PHP_INT_MAX );
-		add_action( 'tec_admin_headers_about_to_be_sent', [ $this, 'redirect_tec_pages_to_guided_setup' ] ); }
+		add_action( 'tec_admin_headers_about_to_be_sent', [ $this, 'redirect_tec_pages_to_guided_setup' ] );
+	}
+
+	/**
+	 * Remove the filter hooks.
+	 *
+	 * @since 6.8.4
+	 */
+	public function remove_filters(): void {
```

Comment:
This is just organizational - moved w/o changing aside from line 96 where we remove the new one